walkthroughs/howto-k8s-timeout-policy/colorapp/app.py

```diff
@@ -0,0 +1,31 @@
+import os
+import config
+import time
+from flask import Flask, request
+from aws_xray_sdk.core import patch_all, xray_recorder
+from aws_xray_sdk.ext.flask.middleware import XRayMiddleware
+
+app = Flask(__name__)
+
+xray_recorder.configure(
+    context_missing='LOG_ERROR',
+    service=config.XRAY_APP_NAME,
+)
+patch_all()
+
+XRayMiddleware(app, xray_recorder)
+
+@app.route('/ping')
+def ping():
+    return 'Pong'
+
+@app.route('/')
+def color():
+    print('----------------')
+    print(request.headers)
+    print('----------------')
+    time.sleep(45)
```

Comment:
It'd be good to read the timeout value from application env so it can be tuned if someone wants tune the timeouts at virtualnode

Reply:
Sure, made the changes